{{cookiecutter.project_slug}}/{{cookiecutter.package_name}}/settings.py

```diff
@@ -7,25 +7,12 @@
 # database ports go into localsettings.py.  May your hear turn purple if you
 # ever put personal settings into this file or into developmentsettings.py!
 
-import logging
 import os
-import tempfile
 
-# Set matplotlib defaults.
-# Uncomment this when using lizard-map.
-# import matplotlib
-# # Force matplotlib to not use any Xwindows backend.
-# matplotlib.use('Agg')
-# import lizard_map.matplotlib_settings
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+# In older projects, this setting is called BUILDOUT_DIR
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
```

Comment:
De join met `..` zie ik niet gebeuren. Ik zou als BASE_DIR die `..` juist pakken en niet de settings dir.

Reply:
Hij doet 2x `os.path.dirname`, wat daarop neerkomt. Komt uit de django template

Reply:
Ah! Dat had ik niet gezien.

Dan wel het commentaar twee regels erboven aanpassen, dat je '..' moet gebruiken in je join.